_pages/amitbasuri.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: amitbasuri
+pageUrl: https://amitbasuri.github.io/amitbasuri
```

Comment:
@amitbasuri you don't need `https://` -- just remove that part and you should be all set!